src/screens/VisualTests/VisualTests.stories.tsx

```diff
@@ -150,7 +151,8 @@ const withBuilds = ({
   lastBuildOnBranch?: LastBuildOnBranchBuildFieldsFragment;
   userCanReview?: boolean;
 }) => {
-  return withGraphQLQueryResult(QueryBuild, {
+  // FIXME -- I'm not sure why I have to mess with the types in this way
+  return withGraphQLQueryResult<typeof QueryBuild, VariablesOf<typeof QueryBuild>>(QueryBuild, {
```

Comment:
@kasperpeulen this feels like the same problem that I don't really understand with the `playAll` thing. The previous type for `withGraphQLQueryResult` wouldn't work because it seemed to "lose" the variables that are bound into a specific type instance of the `TypedDocumentNode`.

The above works but this type it doesn't seem to be able to infer it for us so it's very awkward to use. Maybe we can take a look together?

Reply:
I don't truly understand this issue, but just changing the function definition to this:

`function withGraphQLQueryResult<TQuery extends TypedDocumentNode<any, any>>(`

Make the inference in usages works properly again. Might be a TS bug.

Reply:
OK, that's good enough for me; hardly the most crictical piece of code.

Reply:
Thanks!